armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -261,8 +261,9 @@ def test_width(self):
         for ring, power in zip(range(1, 8), range(10, 80, 10)):
             aList = assemsByRing[ring]
             for a in aList:
+                sf = a.getSymmetryFactor()  # center assembly is only 1/3rd in the core
                 for b in a:
-                    b.p.power = power
+                    b.p.power = power / sf
```

Comment:
are there not other params like power. cant we handle all volume integrated params the same??